flexget/plugins/filter/crossmatch.py

```diff
@@ -99,10 +102,17 @@ def entry_intersects(self, e1, e2, fields=None):
             log.trace('v1: %r' % v1)
             log.trace('v2: %r' % v2)
 
-            if v1 == v2:
-                common_fields.append(field)
+            if config.get('fuzzy', {}):
+                if v2 in v1:
```

Comment:
I'm not sure if makes sense the other way.  v2 is the crossmatch input (what we're searching for), v1 is the source input (RSS for example).  If the RSS has an entry of "quick brown fox jumped over the lazy dog" and your searching for "jumped over"  v2 in v1 works.

However, I don't think it would make sense to accept an RSS entry of "Brown Fox" when searching for "quick brown fox jumped over the lazy dog"...  Right?